client/src/AppBundle/Service/Client/Internal/ReportApi.php

```diff
@@ -7,6 +7,8 @@
 use AppBundle\Entity\Report\Report;
 use AppBundle\Entity\User;
 use AppBundle\Event\ReportSubmittedEvent;
+use AppBundle\Event\ReportUnsubmittedEvent;
+use AppBundle\Event\ReportResubmittedEvent;
```

Comment:
Is this being used?

Reply:
Nope, good spot, twas a hangover from when i was splitting submission into submission and resubmission but that approach created way too much duplicated code..

Reply:
I've done this a couple of times with other events - end up looking at what I've made and its basically the same as another event 😂